frontend/src/components/Menu/Menu.tsx

```diff
@@ -4,10 +4,12 @@ import styled from "styled-components";
 import CardTitle from "../CardTitle/CardTitle";
 import {VerticalSpace} from "../Shared/VerticalSpace";
 import {Centered} from "../Shared/Centered";
+import {Link} from "react-router-dom";
 
 type MenuButtonsProps = {
     readonly textName: string;
     readonly onClick: () => void;
+    readonly onLink: string;
```

Comment:
To jest to co mówiłem, ja bym z tym uważał itak nie robił. Określasz buttony z zewnątrz, a to jest przecież Menu, a nie jakiś kontener na buttony na liście. I teraz każda zmiana wymaga od Ciebie właśnie zmian w czymś co jest publiczne (określonych propsach menu buttona). Nie musisz nad tym pracować w tym PR, ale komentowałem coś podobnego Pawłowi raczej rozbudowanie:

https://github.com/nowakprojects/CodersCamp2020.Project.FullStack-Node-React.TableSoccerTournaments/pull/107#discussion_r602708470

Reply:
![image](https://user-images.githubusercontent.com/41823689/112756454-8c5b2b80-8fe5-11eb-83a5-f41a21047f8b.png)
A dodanie osobnego componentu? który jest HomeMenu i korzysta z Menu?

Reply:
Niby można, tylko po co? Wtedy musisz testować  - tak patrząc dwa razy to samo. W Menu, że Ci pokazuje buttony jakie przekażesz. A w HomeMenu, że pokazuje te buttony, które musi mieć HomeMenu. Może się to przyda, ale jak dla mnie lepiej robić najpierw rzeczy specyficzne, a ewentualnie wyciągnać z nich jak zacznie się wykształcać jakiś powtarzający wzorzec i duplikacja. To taka zasada YAGNI.

Reply:
Dobra, bo już wysłałem zmiany. Zrobię merge i jak coś to w przyszłym PR

Będę pamiętał o tym w przyszłości.
Nie wiedziałem, że ma to jakiś sens, ale racja. 
![image](https://user-images.githubusercontent.com/41823689/112757057-45226a00-8fe8-11eb-9c2b-77e0185c6e0e.png)